seshdash/models.py

```diff
@@ -257,6 +257,20 @@ class Daily_Data_Point(models.Model):
         "daily_power_cons_pv": "W",
         "daily_power_consumption_total": "W",
         "daily_pv_yield": "W",
+        "soc":"%",
```

Comment:
These need to go in the correct model.
This is the daily data point model which is for daily aggregates. Use the BoM data model for this.